include/seastar/core/reactor.hh

```diff
@@ -502,6 +502,8 @@ public:
     future<> touch_directory(std::string_view name, file_permissions permissions = file_permissions::default_dir_permissions) noexcept;
     future<std::optional<directory_entry_type>>  file_type(std::string_view name, follow_symlink = follow_symlink::yes) noexcept;
     future<stat_data> file_stat(std::string_view pathname, follow_symlink) noexcept;
+    future<> change_ownership_of_file(std::string_view filepath, uint64_t groupid) noexcept;
```

Comment:
i'd suggest change the name to `chown()` to be consistent with its cousin of `chmod()`.

Reply:
and, in addition to the gid, how about exposing the uid to the caller?

Reply:
yea, that sounds good.

Reply:
I agree, it's already bad enough that Seastar has its own versions of syscalls and users need to learn how to use them and how their parameters and behavior differ from the traditional Unix ones - let's not make it even more difficult by inventing new names for them. If this is a thin wrapper over chown(), let's call it chown.

Reply:
We have a tradition of using more readable names.